odoo/models.py

```diff
@@ -6461,6 +6461,14 @@ def diff(self, other, force=False):
 
         return result
 
+    @classmethod
+    def _get_placeholder_filename(cls, field=None):
```

Comment:
tbh, I just restored the same api from v14.

If I start to change api, I will prefere to have False as defaut value of Binary.placeholder() and so be able to do:

```python
filename = get_placeholder()
placeholder(filename)

```

and 

```python
    @staticmethod
    def placeholder(image=False):
        if not image:
            image = 'placeholder.png'
```

instead of:

```python
filename = get_placeholder()
if filename:
    placeholder(filename)
else:
    placeholder()
```

and have placeholder.png duplicated 2 times in code

Reply:
Yeah that makes sense.